index.html

```diff
@@ -116,50 +100,100 @@ <h5 class="card-title text-center">Cicerone</h5>
                 </div>
             </div>
         </div>
-
+        <br>
         <h3 class="text-center wdwd">Our Domains</h3>
         <div class="row domain">
 
             <div class="col-lg-6 col-md-6 col-sm-12 rel text-center">
 
-                <img src="assets/home/web.png" class="abs">
-                <div class="web">
-                    <p class=" text-center">Web Development</p>
+                <img src="assets/home/human.png" class="abs">
+                <div class="adv">
+                    <br>
+                    <p class="text-center">Advanced Artificial Intelligence</p>
                 </div>
             </div>
             <div class="col-lg-6 col-md-6 col-sm-12  text-center rel">
 
                 <img src="assets/home/cyber.png" class="abs">
                 <div class="cyber">
-
+                    <br>
                     <p class=" text-center">Cyber Security</p>
                 </div>
             </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 text-center rel">
+
+                <img src="assets/home/ml.png" class="abs">
+                <div class="app">
+                    <br>
```

Comment:
Please check the syntax for `br` tag. It is a self closing tag.

Reply:
I haven't provided </br> anywhere, can you recheck?